app/src/main/java/com/practicum/playlistmaker/ui/track/TrackAdapter.kt

```diff
@@ -0,0 +1,39 @@
+package com.practicum.playlistmaker.ui.track
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.practicum.playlistmaker.R
+import com.practicum.playlistmaker.domain.search.models.Track
+
+class TrackAdapter(private var clickListener: LocationClickListener?): RecyclerView.Adapter<TrackViewHolder>() {
+    fun interface LocationClickListener {
+        fun onLocationClick(track: Track)
+    }
+
+    private var listTrack = ArrayList<Track>()
```

Comment:
```suggestion
    private val listTrack = ArrayList<Track>()
```
Сама ссылка же нигде не переприсваивается. Значит можно сделать val